lib/vault.ts

```diff
@@ -582,10 +582,18 @@ export async function deleteDraftFormResponses(formID: string) {
       responsesDeleted: accumulatedResponses.length,
     };
   } catch (error) {
-    logMessage.error(
-      `Failed to delete form responses from the Vault during publishing for form ${formID}.`
-    );
-    logMessage.error((error as Error).message);
+    if (error instanceof TemplateAlreadyPublishedError) {
+      // Log a warning if the form is published
+      logMessage.warn(
```

Comment:
Updated to just warn we don't need to error out here --- just let UI move along to the published page.